lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -143,6 +143,18 @@ def test_observed_dl1_validity(observed_dl1_files):
     )
     np.testing.assert_allclose(dl1_df["dragon_time"], dl1_df["trigger_time"])
 
+@pytest.mark.private_data
+def test_dvr_file_validity(observed_dl1_files):
```

Comment:
shouldn't the dvr have as input a R0 data file?

Reply:
The script calculates the pixel masks from the dl1 images, stores it in an hdf5 output file and the actual dvr is performed by a C++ program reading the pixel masks and applying them to the zfits files.

The C++ program is here:
https://gitlab.cta-observatory.org/cta-array-elements/lst/subsystems/daq/offline_gain_selection